include/mlio/image_reader.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You
+ * may not use this file except in compliance with the License. A copy of
+ * the License is located at
+ *
+ *      http://aws.amazon.com/apache2.0/
+ *
+ * or in the "license" file accompanying this file. This file is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+ * ANY KIND, either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+#pragma once
+
+#include <opencv4/opencv2/core/mat.hpp>
```

Comment:
It is because the library has not been added using cmake. It will build once this is approved and rebased: https://github.com/rizwangilani/ml-io/pull/2

Reply:
I manually added the library, but it still fails, because you are explicitly including opencv header files in `image_reader.h`.

Reply:
got it, thanks!